docs/examples/rag_azuresearch.ipynb

```diff
@@ -0,0 +1,737 @@
+       "│ ## STATE OF AI REPORT.                                                                                          │\n",
+       "│                                                                                                                 │\n",
+       "│ October 10, 2024                                                                                                │\n",
+       "│                                                                                                                 │\n",
+       "│ Nathan Benaich                                                                                                  │\n",
+       "│                                                                                                                 │\n",
+       "│ AIR STREET CAPITAL.                                                                                             │\n",
+       "│                                                                                                                 │\n",
+       "│ ## About the authors                                                                                            │\n",
+       "│                                                                                                                 │\n",
+       "│ Nathan Benaich                                                                                                  │\n",
+       "│                                                                                                                 │\n",
+       "│ <!-- image -->                                                                                                  │\n",
+       "│                                                                                                                 │\n",
+       "│ Nathan is the General Partner of Air Street Capital , a venture capital firm investing in AI-first companies.   │\n",
+       "│ He runs the Research and Applied AI Summit (RAAIS), the RAAIS Foundation (funding open-source AI projects), AI  │\n",
+       "│ communities in the US and Europe, and Spinout.fyi (improving university spinout creation). He studied biology   │\n",
+       "│ at Williams College and earned a PhD...                                                                         │\n",
+       "╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from rich.console import Console\n",
+    "from rich.panel import Panel\n",
+    "from docling.document_converter import DocumentConverter\n",
+    "\n",
+    "console = Console()\n",
+    "\n",
+    "source_url = \"https://ignite2024demo.blob.core.windows.net/state-of-ai-2024/State of AI Report 2024.pdf\"\n",
```

Comment:
1. I see this is 212 pages of PDF. If the size was explicitly meant to be in that range, it could help to include a quick headsup to the user explaining that and that they would be parsing 200+ pages of PDF. From my point of view a shorter document could would work here.
2. If we stick to this document, I would still URL-encode the URL.